resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -0,0 +1,274 @@
+      <alias value="Gebruiksperiode" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22660" />
+        <comment value="PeriodOfUse" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUseDuration">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="periodOfUseDuration" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval-Duration" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUseDuration.value[x]">
+      <path value="MedicationDispense.extension.value[x]" />
+      <short value="PeriodOfUse / Duration" />
+      <definition value="**Duration**: The intended duration of use. E.g. 5 days or 8 weeks. It is not allowed to indicate the duration in months, because different months have a variable duration in days." />
+      <alias value="Gebruiksperiode" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22660" />
+        <comment value="PeriodOfUse" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.modifierExtension">
+      <path value="MedicationDispense.modifierExtension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType">
+      <path value="MedicationDispense.modifierExtension" />
+      <sliceName value="stopType" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+      </type>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType.value[x]">
+      <path value="MedicationDispense.modifierExtension.value[x]" />
+      <short value="AdministrationAgreementStopType" />
+      <definition value="Stop type, the manner in which this medication is discontinued (temporary or definitive)." />
+      <alias value="ToedieningsafspraakStopType" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22498" />
+        <comment value="AdministrationAgreementStopType" />
+      </mapping>
+    </element>

```

Comment:
Excel mapping says: "Plus uitleg over gebruik MedicationDispense.statusReason. Dat mag samen met deze extensie maar vervangt deze extensie niet."

Don't see that here (or at .statusReason for that matter)

Reply:
Nor in ext-StopType itself. ext-StopType talks about .status though. Don't know if that's whats intended

Reply:
Relates to https://github.com/Nictiz/Nictiz-R4-zib2020/issues/114 and I made a to do for my self to improve this.